src/ucp/wireup/wireup.c

```diff
@@ -1004,6 +1004,19 @@ ucp_wireup_ep_lane_set_next_ep(ucp_ep_h ep, ucp_lane_index_t lane,
                               ucp_ep_get_rsc_index(ep, lane));
 }
 
+static int ucp_wireup_activate_worker_iface(ucp_worker_iface_t *wiface,
+                                            ucp_ep_h ep, ucp_lane_index_t lane)
+{
+    /* Activate worker iface if: a) new protocol selection logic is disabled; or
+     * b) stream support is requested, because stream API does not support new
+     * protocol selection logic; or c) lane is used for checking a connection
+     * state; or d) the endpoint is a mem-type ep. */
+    return !wiface->worker->context->config.ext.proto_enable ||
```

Comment:
maybe helper var for context?

Reply:
Added in https://github.com/openucx/ucx/pull/8986/commits/d3fce5173f95272b332ec2688ec80b787a947e4b